src/vm/eventtrace.cpp

```diff
@@ -6680,16 +6679,15 @@ VOID ETW::MethodLog::SendMethodILToNativeMapEvent(MethodDesc * pMethodDesc, DWOR
     // of 64K
     const USHORT kMapEntriesMax = 7000;
 
+    if (g_pDebugInterface == NULL)
```

Comment:
(same thing as above, this deserves an assert or a revert to prevent accidental loss of functionality. I'll stop calling it out)